ACPs/77-reinventing-subnets/README.md

```diff
@@ -132,37 +209,55 @@ Since there are no `EndTime`s enforced by the P-Chain, Subnets must use this tra
+- `weight` is the new `weight` of the validator
+
+### Removing Subnet Validators
+
+#### SetSubnetValidatorWeightTx (Weight = 0)
+
+Issuing a `SetSubnetValidatorWeightTx` with `Weight` of `0` will remove the Subnet Validator from the Subnet's Validator set.
+
+All state related to the Subnet Validator being removed will be removed from the P-Chain's active state (BLS key, NodeID etc). This validator can be newly added to the Subnet's validator set using the `RegisterSubnetValidatorTx` flow.
 
 #### ExitValidatorSetTx
 
 Subnet Validators can use `ExitValidatorSetTx` to exit the validator set without interacting with the Subnet. An Ed25519 Signature is required using the Subnet Validator's Ed25519 private key for this transaction to be valid. Remaining $AVAX in the Subnet Validator's `Balance` will be issued to the `ChangeOwner` defined when adding this validator to the validator set.
 
-This transaction can be issued on the P-Chain without explicit consent of the Subnet. It is expected that validators should be removed from the Subnet's validator set through a `SetSubnetValidatorWeightTx` with weight `0` by initiating Subnet Validator removal on the Subnet. However, the ability to exit a Subnet Validator set is critical for censorship-resistance and/or failed Subnets. If a validator ever wishes to stop participating in Subnet consensus, they will be able to do so through this transaction. This is enforced on the P-Chain to prevent Subnets from locking Validators into participating in consensus indefinitely.
+This transaction can be issued on the P-Chain without explicit consent of the Subnet. It is expected that validators should be removed from the Subnet's validator set through a `SetSubnetValidatorWeightTx` with weight `0` by initiating Subnet Validator removal on the Subnet. However, the ability to exit a Subnet Validator set is critical for censorship-resistance and/or failed Subnets. If a validator ever wishes to stop participating in Subnet consensus, they will be able to do so through this transaction. This is enforced on the P-Chain to prevent Subnets from locking Validators into participating in consensus indefinitely. Note that this does not modify a Subnet's total staking weight, this transaction moves the Validator from Active to Inactive.
```

Comment:
Wanted to ask for clarification on `Note that this does not modify a Subnet's total staking weight, this transaction moves the Validator from Active to Inactive.`

because above it says `Remaining $AVAX in the Subnet Validator's `Balance` will be issued to the `ChangeOwner`, so can't the left over avax be removed from the Subnet's total staking weight?

Reply:
nvm I mixed up weight and balanace. IIUC, "total staking weight" is referring to the Subnet's defined staking calculations, and "balance" that is moved to ChangeOwner is referred to the P-Chain AVAX balance.